Sources/ArcGISToolkit/Components/Offline/OfflineMapAreaListItemView.swift

```diff
@@ -122,6 +122,11 @@ struct OfflineMapAreaListItemView<Model: OfflineMapAreaListItemInfo, TrailingCon
                     Image(systemName: model.statusSystemImage)
                 }
                 Text(model.statusText)
+                if model.sizeInBytes > 0 && !model.isDownloaded && model.allowsDownload {
+                    Divider()
+                        .frame(height: 12)
+                    Text(model.sizeText)
```

Comment:
```suggestion
                    Text(model.size, format: .byteCount(style: .file))
```
We usually use Text format parameter, so it only needs to be recomputed when needed. (Phil's [comment](https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/617#discussion_r2089939731)) Apply together with the next suggestion.